<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Главное меню</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="1" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Работа с клиентом</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="record_window">
          <property name="text">
           <string>Запись</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="list_of_client">
          <property name="text">
           <string>Список клиентов</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="client_card">
          <property name="text">
           <string>Карточка клиента</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="1" rowspan="3">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Сервис</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="types_of_gearbox">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Виды коробок передач</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="service_groups">
         <property name="text">
          <string>Группы услуг</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="list_of_parts">
         <property name="text">
          <string>Список запчастей</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="list_of_services">
         <property name="text">
          <string>Список услуг</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="group_of_parts">
         <property name="text">
          <string>Группы запчастей</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="engine_types">
         <property name="text">
          <string>Типы двигателей</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Управление складом</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="stock_balance">
         <property name="text">
          <string>Баланс на складе</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="arrival_of_parts">
         <property name="text">
          <string>Приход запчастей</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="list_of_orders">
         <property name="text">
          <string>Список заказов</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>420</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
